lib/web/forms/registration.go

```diff
@@ -0,0 +1,60 @@
+package webforms
+
+import (
+	"errors"
+
+	"go-google-scraper-challenge/constants"
+	"go-google-scraper-challenge/lib/models"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/go-ozzo/ozzo-validation/v4/is"
+)
+
+type RegistrationForm struct {
+	Email                string `form:"email"`
+	Password             string `form:"password"`
+	PasswordConfirmation string `form:"password_confirmation"`
+}
+
+func (f RegistrationForm) Validate() (valid bool, err error) {
```

Comment:
Should we use a pointer receiver here?
```suggestion
func (f *RegistrationForm) Validate() (valid bool, err error) {
```

Same for other below methods and `lib/web/controllers/users.go` as well 🙇

Reply:
@hoangmirs I don't think the pointer receiver is necessary as there is no mutation within the function and I also use the value receiver for other forms including the form for API too, could you point me out why should we use it here? 😅

Reply:
Ah, then it's ok for the form objects. But for controllers, as we used pointer receiver on some controllers already, should we apply it for other controllers for consistency?

Reply:
Oh nice catch, fixed in [3a0d448](https://github.com/carryall/go-google-scraper-challenge/pull/125/commits/3a0d4484d62946086c632a03c2aec7f9257905c6)